classes/TranslatableBehavior.php

```diff
@@ -356,6 +356,20 @@ public function isTranslateDirty($attribute = null, $locale = null)
         }
     }
 
+    /**
+     * Get the original values of the translated attributes
+     * @param  string $locale
+     * @return array
+     */
+    public function getTranslatableOriginals($locale = null)
+    {
+        if (!$locale) {
+            return $this->translatableOriginals;
+        } else {
+            return $this->translatableOriginals[$locale];
```

Comment:
@mjauvin Would you be able to make this return `null` if someone puts in a locale that doesn't exist?
```suggestion
            return $this->translatableOriginals[$locale] ?? null;
```